integration_tests/bindc_03.py

```diff
@@ -7,14 +7,14 @@
 class ArrayWrapped:
     array: CPtr
 
-@ccall(header="bindc_03b.h")
+@ccall(module="bindc_03b.h")
```

Comment:
I would call it a `header`, I think it's more intuitive what it means.

Reply:
The `@pythoncall(module="py_module")` usage is fine.

Reply:
Got it. I reverted it back to using `header`. I added support both keywords `header` and `module`.